deploy/gcp/main.tf

```diff
@@ -50,7 +50,7 @@ module "tidb-operator" {
   kubeconfig_path               = local.kubeconfig
   tidb_operator_version         = var.tidb_operator_version
   maintenance_window_start_time = var.maintenance_window_start_time
-  operator_helm_values          = var.operator_helm_values
+  operator_helm_values          = var.operator_helm_values == "" ? "" : file(var.operator_helm_values)
```

Comment:
inline yaml values is sometimes convenient, short and no external file required. How about adding a new variable for file support?

```
operator_helm_values = <<EOF
operatorImage: quay.io/cofyc/tidb-operator:canary
imagePullPolicy: Always
EOF
```

Reply:
OK, I thought it was designed for external values file, will revert the change.